molecule/generic_files/Dockerfile.j2

```diff
@@ -8,8 +8,8 @@ FROM {{ item.image }}
 {% endif %}
 
 RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y apt-utils python sudo bash ca-certificates systemd dbus lsb-release && apt-get clean; \
-    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python3 sudo python3-devel python3-dnf bash redhat-lsb-core && dnf clean all; \
-    elif [ $(command -v yum) ]; then yum makecache fast && yum install -y python sudo yum-plugin-ovl bash redhat-lsb-core && sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf && yum clean all; \
+    elif [ $(command -v dnf) ]; then dnf makecache && dnf install -y redhat-lsb-core && dnf install -y python3 sudo python3-devel python3-dnf bash && dnf clean all; \
+    elif [ $(command -v yum) ]; then yum makecache fast && yum install -y redhat-lsb-core && yum install -y python sudo yum-plugin-ovl bash && sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf && yum clean all; \
```

Comment:
I wonder if we actually need lsb-core for our tests. I think I already removed it in another role where it also caused issues and since i couldn't find any changes in the results.
@dverhelst I think you initially wrote this template. Any idea why it's here?

Reply:
Tried the test without it and it fails, didn't go into digging what was wrong though.
I can do so if you like.

Reply:
In case of the other role I was able to replace a check for the distro version with one without lsb, but I quickly checked and it's not included here. It's not that important, I'll merge.